pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -458,23 +459,112 @@ func (m *NamespaceOption) GetHostIpc() bool {
 	return false
 }
 
+// SandboxSecurityContext holds security configuration that will be applied to a sandbox.
+type SandboxSecurityContext struct {
+	// The configurations for the sandbox's namespaces.
+	// This will be used only if the PodSandbox uses namespace for isolation.
+	NamespaceOptions *NamespaceOption `protobuf:"bytes,1,opt,name=namespace_options,json=namespaceOptions" json:"namespace_options,omitempty"`
+	// Optional SELinux context to be applied. Only for sandbox. Containers should
+	// confirm to SecurityContext.SelinuxOptions.
+	SelinuxOptions *SELinuxOption `protobuf:"bytes,2,opt,name=selinux_options,json=selinuxOptions" json:"selinux_options,omitempty"`
+	// The UID to run the entrypoint of the container process.
+	// Defaults to user specified in image metadata if unspecified.
+	// Only for sandbox. Containers should confirm to SecurityContext.RunAsUser.
+	RunAsUser *int64 `protobuf:"varint,3,opt,name=run_as_user,json=runAsUser" json:"run_as_user,omitempty"`
+	// Indicates that the container must run as a non-root user.
+	// If true, the runtime should validate the image at runtime to ensure that it
+	// does not run as UID 0 (root) and fail to start the container if it does.
+	// If unset or false, no such validation will be performed.
+	// Only for sandbox. Containers should confirm to SecurityContext.RunAsNonRoot.
+	RunAsNonRoot *bool `protobuf:"varint,4,opt,name=run_as_non_root,json=runAsNonRoot" json:"run_as_non_root,omitempty"`
+	// If set, the root filesystem of the container is read-only.
+	ReadonlyRootfs *bool `protobuf:"varint,5,opt,name=readonly_rootfs,json=readonlyRootfs" json:"readonly_rootfs,omitempty"`
+	// A list of groups applied to the first process run in the sandbox, in addition
+	// to the container's primary GID.  If unspecified, no groups will be added.
+	SupplementalGroups []int64 `protobuf:"varint,6,rep,name=supplemental_groups,json=supplementalGroups" json:"supplemental_groups,omitempty"`
+	// A special supplemental group that applies to the sandbox.
+	// Some volume types allow the Kubelet to change the ownership of that volume
+	// to be owned by the pod:
+	//
+	// 1. The owning GID will be the FSGroup
+	// 2. The setgid bit is set (new files created in the volume will be owned by FSGroup)
```

Comment:
Note to self, make sure this is accurate after fix to setting setgid bit on files is merged.